<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderView</class>
 <widget class="QWidget" name="FolderView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="directoryPathLabel">
       <property name="text">
        <string>[path]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="IconButton" name="openButton">
       <property name="text">
        <string>openbtn</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="IconButton" name="settingsButton">
       <property name="text">
        <string>settingsbtn</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FolderGridView" name="thumbnailGrid"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FolderGridView</class>
   <extends>QGraphicsView</extends>
   <header>gui/folderview/foldergridview.h</header>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QLabel</extends>
   <header>gui/customwidgets/iconbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
